### PR TITLE
Add methods contributions to metadata

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Here's a preliminary stab at a list of authors for the methods paper (currently unordered). See #948 and we will likely have another author being added as well.
I took the authors from the poster I presented at CSHL Biological Data Science back in November and added @mprobson (who has done a lot of behind-the-scenes work helping me manage a project of this scale in git, whereas before I'm not sure i had ever had a merge conflict!)

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by agitter — APPROVED** [review]
Looks good to me.  Did you discuss with @cgreene already or should we wait for him to review before merging?

**Comment by rando2** [comment]
@agitter I didn't go over this with @cgreene, I know he's planning to take a look tomorrow afternoon. This one the author list is mostly copied from the poster.

**Review by cgreene — APPROVED** [review]
🚀

**Pull request merged**
